index.html

```diff
@@ -1,6 +1,9 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.0/jquery.min.js"></script>
+  //had to look this one up//
```

Comment:
be careful on comment syntax. For JS, comments are `//` but in html, comments are `<!-- -->`